libcodechecker/analyze/analysis_manager.py

```diff
@@ -274,7 +275,8 @@ def check(check_data):
 
 
 def start_workers(actions, context, analyzer_config_map,
-                  jobs, output_path, skip_handler, metadata):
+                  jobs, output_path, skip_handler, metadata,
+                  ctu_collect, ctu_analyze, ctu_dir, ctu_func_map_cmd):
```

Comment:
Adding this many new parameters to the function doesn't feel the best. Perhaps we could somehow use the extended `config_handler` classes in `libcodechecker.analyze` instead?